gcn/notices/icecube/LVK_nu_track_search.example.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "https://gcn.nasa.gov/schema/gcn/notices/icecube/LVK_nu_track_search.schema.json",
+  "type": "IceCube LVK Alert Nu Track Search",
+  "event_name": ["MS230427r"],
+  "id": ["Initial", "Sequence: 2"],
```

Comment:
Are these trigger IDs for each instrument, LVK and nu, doesn't look like?

Reply:
these are identifiers from the LVK alert, letting people know which LVK alert, and at what level (initial, preliminary, etc) it was performed at

Reply:
It doesn't seem correct field, ID is ID number of event as observed by instrument. I have made some changes, please check them

Reply:
We have added a new core schema "follow_up".  We've also removed "id_source" from event.  Hopefully this should be a clearly way to indicate that a reporter is reporting on the follow-up of a reference trigger.  You can cite a GCN circular or notice rather than "initial", "sequence 2".  We're working on documentation on how to reference a specific notice by topic name and offset number.

Reply:
Changed event_name to ref_ID from FollowUp in core.